src/comb/template.clj

```diff
@@ -57,6 +57,6 @@
   ([source]
      (eval source {}))
   ([source bindings]
-     (let [keys (map (comp symbol name) (keys bindings))
+     (let [keys (apply vector (map (comp keyword name) (keys bindings)))
```

Comment:
I pushed a new version. Previous version work with `1.9.0-alpha14`, but not work with `1.2.1`, new version support both two version.